cypress/integration/SellingOrder.spec.ts

```diff
@@ -20,7 +20,7 @@ describe('Sell Order', () => {
 
     const orders = Orderbook.list(OrderType.SELL);
     orders.countIs(4);
-    makeScreenshots("new-order");
+    makeScreenshots('new-order');
```

Comment:
This was caused by misconfigured linter settings. Could you fix them right away?

in tsconfig.json please add:
```
    "cypress/**/*.ts",
    "cypress/**/*.tsx",
```

in include list. Then run `yarn test:fix`. I would do this on separate branch but this will just cause a lot of conflicts.